electron/main.ts

```diff
@@ -224,12 +225,12 @@ if (!gotTheLock) {
     })
     if (!app.isDefaultProtocolClient('heroic')) {
       if (app.setAsDefaultProtocolClient('heroic')) {
-        console.log('Registered protocol with OS.')
+        Logger.info({message: 'Registered protocol with OS', service: 'ipcMain::registerHeroicProtocol'})
       } else {
-        console.log('Failed to register protocol with OS.')
+        Logger.warn({message: 'Failed to register protocol with OS', service: 'ipcMain::registerHeroicProtocol'})
       }
     } else {
-      console.log('Protocol already registered.')
+      Logger.info({message: 'Protocol already registered with OS', service: 'ipcMain::registerHeroicProtocol'})
```

Comment:
I think is unecessary and doesn't help anything since it will just flood the logs with the same message everytime we open heroic.
Logging success and error is enough.
Since the Logs will be used to help Debug, the most import part are the errors, not the confirmation that something was already done 1000 times :P

Reply:
Should I remove this else as it isn't doing anything then?

Reply:
Left a comment in it instead